### PR TITLE
edits to vpgl

### PR DESCRIPTION
**Pull request by wtreible**

### CODE REVIEW

File: pyvpgl.cxx [hunk 1]
```diff
@@ -112,12 +175,100 @@ void wrap_vpgl(py::module &m)
...
+        vgl_point_2d<double> img_point = R.project(world_point);
+        return std::make_tuple(img_point.x(), img_point.y());
+      }
+    );
 
   m.def("read_local_rational_camera",
         [](std::string const& fname){return read_local_rational_camera<double>(fname);},
         py::return_value_policy::take_ownership);
 
+  /*  Looks like someone added these already - Wayne
```
Comment (decrispell) [inline pyvpgl.cxx]: Can you remove this chunk of commented out code?

File: pyvpgl.cxx [hunk 2]
```diff
@@ -112,12 +175,55 @@ void wrap_vpgl(py::module &m)
...
+        return std::make_tuple(img_point.x(), img_point.y());
+      }
+    );
 
   m.def("read_local_rational_camera",
         [](std::string const& fname){return read_local_rational_camera<double>(fname);},
         py::return_value_policy::take_ownership);
 
+  // Compute Affine Camera from Local Rational Camera
+  m.def("compute_affine_from_local_rational",
```
Comment (decrispell) [inline pyvpgl.cxx]: Should this function go into vxl, and then be wrapped?

Comment (wtreible) [reply pyvpgl.cxx]: I'm not sure. It is a similar case of this being essentially a copy from the old-style python contrib that was probably a one-off process. I could add it to the local rational cam vxl code if you think that's where it should go though.

**Comment by decrispell** [comment]
A lot of what @wtreible did for this PR had been repeated in the meantime.  I rebased this branch onto current master and sorted out the conflicts.

File: pyvpgl.cxx (hunk 2 above)
Comment (decrispell) [reply pyvpgl.cxx]: @wtreible This functionality is now available in vpgl_algo and wrapped in pyvpgl_algo

**Pull request merged**